### PR TITLE
Renamed shared agent.conf file to shared.conf

### PR DESCRIPTION
**Pull request by cborla**
|Related issue|
|---|
|#7977|

## Description
Renamed shared agent.conf file to shared.conf in Documentation 

## Checks
- [x] It compiles without warnings.
- [x] Spelling and grammar. 
- [x] Used impersonal speech. 
- [x] Used uppercase only on nouns. 
- [ ] Updated the `redirect.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).

## Note to the reviewer

This PR includes changes to the `redirect.js` script that need to be included in all production branches.

### CODE REVIEW

File: source/user-manual/reference/centralized-configuration.rst [hunk 1]
```diff
@@ -246,11 +246,11 @@ The following is an example of how a centralized configuration can be done.
...
+It's important to understand which configuration file takes precedence between ``shared.conf`` and ``agent.conf`` when central configuration is used. When central configuration is utilized, the local and the shared configuration are merged, however, the ``agent.conf`` file is read before the shared ``shared.conf`` and the last configuration of any setting will overwrite the previous. Also, if a file path for a particular setting is set in both of the configuration files, both paths will be included in the final configuration.
 
 For example:
 
-Let's say we have this configuration in the ``shared/agent.conf`` file:
+Let's say we have this configuration in the ``shared.conf`` file:
```
Comment (TomasTurina) [inline source/user-manual/reference/centralized-configuration.rst:253]:
```suggestion
Let's say we have this configuration in the ``agent.conf`` file:
```

**Review by TomasTurina — COMMENTED** [review]
Also rename references of file `source/user-manual/reference/ossec-conf/client.rst`.

File: source/user-manual/reference/centralized-configuration.rst (hunk 1 above)
Comment (cborla) [reply source/user-manual/reference/centralized-configuration.rst]: I Agree, done

**Review by TomasTurina — APPROVED** [review]
LGTM!

**Pull request merged**